src/renderers/shared/fiber/ReactChildFiber.js

```diff
@@ -163,6 +168,10 @@ function createFirstChild(returnFiber, existingChild, newChildren, priority) {
 
 // TODO: This API won't work because we'll need to transfer the side-effects of
 // unmounting children to the returnFiber.
-exports.reconcileChildFibers = function(returnFiber : Fiber, currentFirstChild : ?Fiber, newChildren : ReactNodeList, priority : PriorityLevel) : ?Fiber {
+exports.reconcileChildFibers = function(
+  returnFiber : Fiber, 
+  currentFirstChild : ?Fiber, 
+  newChildren : ReactNodeList, 
+  priority : PriorityLevel) : ?Fiber {
```

Comment:
Ok @keyanzhang , let me try that. Thanks. :)
Pushing new patch to the original PR #7230 in a while. Running `grunt test` and `grunt lint` first.